src/Web/WebMVC/libman.json

```diff
@@ -26,9 +26,9 @@
     {
       "library": "jquery-validate@1.19.0",
       "destination": "wwwroot/lib/jquery-validate/",
-      "files": [
-        "jquery.validate.min.js",
-        "jquery.validate.js"
+      "files": [	
```

Comment:
This change just adds trailing spaces, I don't usually care about this, but perhaps just revert the changes in this file!